xml/ha_storage_protection.xml

```diff
@@ -869,25 +931,38 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
    CGroup: /system.slice/sbd.service
            ├─<emphasis role="strong">1859 sbd: inquisitor</emphasis>
 [...]</screen>
-     <para>Use the process ID (in our example, <literal>1859</literal>) and
-      kill the SBD inquisitor:
-     </para>
-     <screen>&prompt.root;<command>kill</command> -9 1859 </screen>
+      </step>
+      <step>
+       <para>Simulate an SBD failure by killing the SBD inquisitor process.
+       In our example, the process ID of the SBD inquisitor is
+         <literal>1859</literal>):</para>
+       <screen>&prompt.root;<command>kill</command> -9 1859 </screen>
+       <para>
+        <remark>taroth 2018-04-28: @DEVs, What is the expected result?</remark>
```

Comment:
The node proactively self-fences. And the other nodes notices the loss of the node and consider it has self-fenced after the stonith-watchdog-timeout.